tests/scipy_stats_test.py

```diff
@@ -564,6 +564,31 @@ def testIssue972(self):
       lsp_stats.norm.cdf(np.full((4,), np.inf, np.float32)),
       check_dtypes=False)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": jtu.format_test_name_suffix("", [shape], [dtype]),
```

Comment:
`jtu.format_test_name_suffix` is usually used when you have a name you need to format, it's strange to see it with an empty string.